app/views/profiles/show.html.erb

```diff
@@ -1,3 +1,6 @@
+<% @title = t('.title', username: @user.display_id) %>
+<% @page_title_only = true %>
```

Comment:
Isn't there any better way to achieve the same result? Maybe `content_for` could be useful in here.

Reply:
There current implementation to change the page title uses this helper:

https://github.com/rubygems/rubygems.org/blob/3b04e8182c773d0d3942e08a5035f5cda623c242/app/helpers/application_helper.rb#L2-L5

https://github.com/rubygems/rubygems.org/blob/3b04e8182c773d0d3942e08a5035f5cda623c242/app/views/layouts/application.html.erb#L4

Makes use of @title to modify the page title, but the problem is that there is no way to only modify page title, it shows the title on the content too here:

https://github.com/rubygems/rubygems.org/blob/3b04e8182c773d0d3942e08a5035f5cda623c242/app/views/layouts/application.html.erb#L107

I introduced @page_title_only so as not to touch other calls. Open to suggestions

Reply:
Changed to manage that logic in page_title helper, now if we define the instance variable @title_for_header_only the title is only for HTML > head > title, and the @title check on the template is as before